frontend/public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>MemoraCar</title>
+    <title>shinyHedgehog</title>
```

Comment:
It'd have been better to use the env key containing the name, not replace on hard-coded value with another